apps/tng-radio-hub/src/Components/pages/links/links.tsx

```diff
@@ -0,0 +1,35 @@
+import LaunchIcon from "@mui/icons-material/Launch";
+import './links.module.css';
+
+
+/* eslint-disable-next-line */
+export interface LinksProps {}
+
+export function Links(props: LinksProps) {
+    return (
+      <div className="links">
+      <a
```

Comment:
Similar comment to that for contacts. You need a type \ data structure for an array of links. Define the data in ts. Then loop over this and feed each one into a presentational component.